app/authorization/server/startup.js

```diff
@@ -140,6 +140,16 @@ Meteor.startup(function() {
 		{ _id: 'view-all-teams', roles: ['admin'] },
 		{ _id: 'remove-closed-livechat-room', roles: ['livechat-manager', 'admin'] },
 		{ _id: 'remove-livechat-department',  roles: ['livechat-manager', 'admin'] },
+
+		// VOIP Permissions
+		// allows to manage voip calls configuration
+		{ _id: 'manage-voip-calls', roles: ['livechat-manager', 'admin'] },
+		// allows to hook on an ongoing call and listen
+		{ _id: 'ghost-voip-calls', roles: ['livechat-manager', 'livechat-monitor'] },
+		// allows to receive a voip call
+		{ _id: 'receive-voip-calls', roles: ['livechat-agent'] },
+		// allows to perform an outgoing voip call
+		{ _id: 'start-voip-calls', roles: ['livechat-agent'] },
```

Comment:
Except for the `start-voip-calls` permission, all of the others above will be part of the EE version. Do you guys remember that?

Reply:
You mean, the `receive-voip-calls` isn't it? The one for incoming calls. Also, the `manage-voip-calls` should also be on CE since that's for admins to manage the voip configuration.

I'll move the other 2 to EE.

Reply:
You are right the only one available in CE is `receive-voip-calls`, I just fixed it in the previous comment. My bad, sorry. Regarding `manage-voip-calls`, we might need to break it down a little more. There'll be `admin` settings and `management` settings, but I just can see one setting here, so I'm a bit confused, to be honest.